### ISSUE TITLE
Startified folds based on multiple columns + column of multi-labels. 

### ISSUE THREAD

**Issue by SoufianeDataFan** [trent-b/iterative-stratification#9]
I have a dataset with a multi-labels column. I want to create stratified folds based not only on the labels column but also on other columns such as age and gender. Is it possible to perform that with `iterstrat`?

**Comment by trent-b** [comment]
Yes, it is possible to encode your data in a way that `iterstrat` may be used with it. For a variable like age, if you have a lot of data, you could [one-hot encode](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.OneHotEncoder.html) age. In the case that you do not have a lot of data, you can bin age (for example, 0-10, 11-20, 20-30, etc.) and then create a binary column for each bin. It's then just a matter of appending your binary label columns with the encoded binary columns.
